tests/n8n/devenv.nix

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
To test that it launches, you can add a `.test.sh` script in this folder and `curl` a health endpoint of some kind. There are lots of examples of this in the repo.